### PR TITLE
update syntaxes

### PR DESCRIPTION
**Pull request by ahgm**
add attribute:
'unsafe_fn'
'deprecated'

### CODE REVIEW

File: syntaxes/v.tmLanguage.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
 		},
 		{
 			"name": "meta.definition.escaped.keyword.v",
-			"match": "((?:@)(?:mut|pub|fn|module|import|const|map|assert|sizeof|type|struct|interface|enum|in|or|switch|case|match|if|else|for|go|goto|defer|return|int|uint|i?(?:8|16|32|64|128)|u?(?:8|16|32|64|128)|f?(?:32|64)|bool|byte|byteptr|voidptr|string|ustring|rune|none))",
+			"match": "((?:@)(?:mut|pub|fn|module|import|const|map|assert|sizeof|type|struct|interface|enum|in|or|match|if|else|for|go|goto|defer|return|i?(?:8|16|nt|64|128)|u?(?:8|16|32|64|128)|f?(?:32|64)|bool|byte|byteptr|voidptr|string|ustring|rune|none))",
```
Comment (Alhadis) [inline syntaxes/v.tmLanguage.json]: Is `@uint` not supported anymore?

Comment (ahgm) [reply syntaxes/v.tmLanguage.json]: yes

Comment (Alhadis) [reply syntaxes/v.tmLanguage.json]: LGTM then.

Comment (ahgm) [reply syntaxes/v.tmLanguage.json]: thanks

**Pull request merged**